test/common.bash

```diff
@@ -26,6 +26,12 @@ builder_ok () {
     #docker_tag_p "${1}:$(ch-run --version |& tr '~+' '--')"
 }
 
+builder_required () {
+    if [[ $CH_BUILDER == none ]]; then
+        skip 'builder required'
+    fi
+}
+
```

Comment:
The sense of this name implies, I think, that a builder is required, and the test will fail if it's not there. But, it just skips.